src/gordon_janitor_gcp/plugins/authority.py

```diff
@@ -144,15 +144,16 @@ def __init__(self, config, crm_client, gce_client, rrset_channel=None,
                 project, instance_filter=instance_filter)
 
     def _create_instance_rrset(self, instance):
+        ip = instance['networkInterfaces'][0]['accessConfigs'][0]['natIP']
```

Comment:
Could this fail? I know that there's a brief period after an instance is created when it hasn't been assigned an IP yet.

Reply:
I took the error message that was there previously on bad data, but I changed the level to warn, because it seems like a more severe indication of a problem with the system if this happens consistently.

Reply:
> i.e. it won't be in the instance data

I think this is what's going to happen. Rarely, but given that we'll be running this every 5 minutes, it might happen a few times a year and annoy us.

> I took the error message that was there previously on bad data, but I changed the level to warn, because it seems like a more severe indication of a problem with the system if this happens consistently.

SGTM!

Reply:
What's the reasoning for only caring about the external IP? I think I know, I just want to make sure I understand.